### DISCUSSION TITLE
I cannot block offensive content

### DISCUSSION THREAD

**Discussion by mika314** [community/community#28218] [Other Feature Feedback, Questions, & Ideas]
See the screenshot

![image](https://user-images.githubusercontent.com/1877406/182657780-d151ea71-c887-43ff-a591-75463f98a1ec.png)

**Comment by jfullstackdev** [comment]
is it the at the `For you` feed ? seems there are related issues like already seeing those starred repos in Recommendations 

let's see whether some Staff members will see this post
